Testing/workflow/DeployUtils.Tests.ps1

```diff
@@ -0,0 +1,201 @@
+# The purpose of this test is to verify that some of the DeployUtils functions, used to publish ScubaGear, are working correctly.
+
+# Suppress PSSA warnings here at the root of the test file.
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSAvoidGlobalVars', '')]
+param()
+
+BeforeAll {
+  # Write-Warning $PSScriptRoot
+  . $PSScriptRoot/../../utils/DeployUtils.ps1
+}
+
+Describe "Check ConfigureScubaGearModule" {
+  It "The manifest file should be altered" {
+    # Get contents of manifest file.
+    $Location = Get-Location
+    $ManifestFilePath = Join-Path -Path $Location -ChildPath "PowerShell/ScubaGear/ScubaGear.psd1"
+    $OriginalContent = Get-Content -Path $ManifestFilePath
+    # ForEach ($Line in $OriginalContent) { Write-Warning $Line }
+    # Setup input parameters.
+    $ModuleFolderPath = Join-Path -Path $Location -ChildPath "/PowerShell/ScubaGear"
+    $ModuleVersion = "9.9.9"
+    $Tag = "help"
+    ConfigureScubaGearModule -ModulePath $ModuleFolderPath -OverrideModuleVersion $ModuleVersion -PrereleaseTag $Tag
+    # Get the new contents of the manifest file
+    $ModifiedContent = Get-Content -Path $ManifestFilePath
+    # ForEach ($Line in $ModifiedContent) { Write-Warning $Line }
+    # Diff the original with the modified
+    $Diff = Compare-Object -ReferenceObject $OriginalContent -DifferenceObject $ModifiedContent
+    # Look for changes for a new module version
+    $ModuleVersionDiff = $Diff | Out-String | Select-String -Pattern $ModuleVersion
+    # There should be only 1 match.
+    ForEach ($Match in $ModuleVersionDiff.Matches) {
+      $Match | Should -Be $ModuleVersion
+    }
+    # Look for changes for a new tag
+    $TagDiff = $Diff | Out-String | Select-String -Pattern $Tag
+    ForEach ($Match in $TagDiff.Matches) {
+      $Match | Should -Be $Tag
+    }
+  }
+}
+
+Describe "Check CreateArrayOfFilePaths" {
+  It "The array should have all the files in it" {
+    # Pick any location, doesn't matter.  For now, get utils folder.
+    $Location = Get-Location
+    $UtilsFolderPath = Join-Path -Path $Location -ChildPath "utils"
+    # For this location, get all the PS files.
+    # This could be any type of file; it doesn't really matter.
+    $Files = Get-ChildItem -Path $UtilsFolderPath -Filter "*.ps1"
+    # Add the files to an array.
+    $ArrayOfPowerShellFiles += $Files
+    # Write-Warning "Length of array is $($ArrayOfPowerShellFiles.Length)"
```

Comment:
```suggestion
```

Reply:
This is test code, and lines like this are for debugging.  It's handy to have these around for the next time I have to debug a problem.  I added a comment noting that this was for debugging, but I left the line there.